ui/packages/atlasmap/src/Views/ColumnMapperView/Actions/EnableJavaClassAction.tsx

```diff
@@ -28,6 +30,7 @@ export const EnableJavaClassAction: FunctionComponent<IEnableJavaClassActionProp
         variant="plain"
         onClick={onEnableClassSearch}
         aria-label="Enable specific Java classes from your previously imported Java archive."
+        data-testid={`enable-specific-java-classes-${id}-button`}
```

Comment:
Example: `{...props}` instead of `data-testid={...}`

Then where this component is used you can simply set the `data-testid` property as you need it